packages/suite/src/views/dashboard/components/AssetsCard/index.tsx

```diff
@@ -70,6 +91,24 @@ const AssetsCard = () => {
     });
     const networks = Object.keys(assets);
 
+    const assetsData: assetType[] = networks
```

Comment:
Maybe worth using `useMemo` here.

Reply:
Realy? with loading data.

```javascript
    const assetsData: assetType[] = useMemo(()=> networks.flatMap(symbol => {
        const network = NETWORKS.find(n => n.symbol === symbol && !n.accountType);
        if (!network) {
            console.error('unknown network');
            return [];
        }

        const assetBalance = assets[symbol].reduce(
            (prev, a) => prev.plus(a.formattedBalance),
            new BigNumber(0),
        );

        const assetFailed = accounts.find(f => f.symbol === network.symbol && f.failed);
        return [{ symbol, network, assetFailed: !!assetFailed, assetBalance }];
    }),networks);
```

Reply:
The second `useMemo` parameter should be an array containing dependencies (`[assets, networks]`). ESLint should suggest the right values. I guess that should fix it.

Reply:
fixed now is ok